repository/spark/operator/operator.yaml

```diff
@@ -7,22 +7,34 @@ appVersion: "2.4.3"
 maintainers:
   - name: Anton Kirillov
     email: akirillov@d2iq.com
-  - name: Uladzimir Krautsou
-    email: ukrautsou.c@d2iq.com
+  - name: Alexander Lembiewski
+    email: alembiyeuski.c@d2iq.com
 url: https://spark.apache.org/
 tasks:
-  - name: deploy
+  - name: deploy-cluster-scoped-resources
     kind: Apply
     spec:
       resources:
         - spark-operator-crds.yaml
         - spark-operator-rbac.yaml
+  - name: deploy-sa
+    kind: Apply
+    spec:
+      resources:
         - spark-operator-serviceaccount.yaml
         - spark-rbac.yaml
         - spark-serviceaccount.yaml
+  - name: deploy-webhook
+    kind: Apply
+    spec:
+      resources:
         - webhook-init-job.yaml
-        - spark-operator-deployment.yaml
         - webhook-service.yaml
+  - name: deploy-spark
```

Comment:
It would be great to disambiguate this name to reflect that it refers to Spark components, e.g. `deploy-spark-components` because we have a step with a similar name.